src/main/main.cc

```diff
@@ -656,18 +660,19 @@ void makedaemon() {
 	fflush(stdout);
 	fflush(stderr);
 	if (pipe(piped)<0) {
-		safs_pretty_syslog(LOG_ERR, "pipe error");
+		safs::log_err("pipe error: {}", strerr(errno));
```

Comment:
In case you want to use it, `log_error_code` also handles `errno`:
```suggestion
		safs::log_error_code(errno, "pipe error");
```